### PR TITLE
fix: reduce number of products from 100 to 50 on lists of products

### PR DESCRIPTION
**Pull request by stephanegigandet**
This should improve performance and reduce the number of requests made to load list of products pages.
Also limits the number of tags in lists of tags, and adds hard limits.

### CODE REVIEW

File: lib/ProductOpener/Config_opff.pm [hunk 1]
```diff
@@ -174,7 +172,31 @@ $flavor = "opff";
 	teolemon
 );
 
-$options{product_type} = "petfood";
+%options = (
+	site_name => "Open Pet Food Facts",
+	product_type => "petfood",
+	og_image_url => "https://world.openpetfoodfacts.org/images/misc/openpetfoodfacts-logo-en.png",
```
Comment (teolemon) [inline lib/ProductOpener/Config_opff.pm:178]: 404

File: lib/ProductOpener/Config_opf.pm [hunk 2]
```diff
@@ -174,7 +172,31 @@ $flavor = "opf";
 	teolemon
 );
 
-$options{product_type} = "product";
+%options = (
+	site_name => "Open Products Facts",
+	product_type => "products",
+	og_image_url => "https://world.openproductsfacts.org/images/misc/openproductsfacts-logo-en.png",
```
Comment (teolemon) [inline lib/ProductOpener/Config_opf.pm:178]: 404

File: lib/ProductOpener/Config_opff.pm (hunk 1 above)
Comment (teolemon) [reply lib/ProductOpener/Config_opff.pm:178]: https://static.openpetfoodfacts.org/images/misc/openfoodfacts-logo-en-356x300.png ?

File: lib/ProductOpener/Config_opf.pm (hunk 2 above)
Comment (teolemon) [reply lib/ProductOpener/Config_opf.pm:178]: https://static.openproductsfoodfacts.org/images/misc/openproductsfacts-logo-en-356x300.png

File: lib/ProductOpener/Config_opff.pm (hunk 1 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Config_opff.pm:178]: We don't want the old logo, we will add the new logo later.

**Review by teolemon — APPROVED** [review]

**Pull request merged**